database/ticket.go

```diff
@@ -43,6 +43,7 @@ const (
 // deliberately kept short because they are duplicated many times in the db.
 type Ticket struct {
 	Hash              string `json:"hsh"`
+	BlockHeight       int64  `json:"bhgt"`
```

Comment:
Can we go with `PurchaseHeight` to make the name slightly more descriptive please